### PR TITLE
RepositorySaverService tests.

### PR DESCRIPTION
**Pull request by materiywakal**

### CODE REVIEW

**Review by ArBond — APPROVED** [review]

File: NugetAnalyzer.BLL.Test/Services/RepositoryServiceTests.cs [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+namespace NugetAnalyzer.BLL.Test.Services
+{
+    [TestFixture(Category = "UnitTests")]
+    public class RepositoryServiceTests
+    {
+        private const int UserId = 1;
+        private const int NullUserId = 0;
+        private readonly Models.Repositories.Repository NullRepository = null;
+        private readonly Models.Repositories.Repository AccurateRepository = new Models.Repositories.Repository
+        {
```
Comment (AntsiferauGodel) [inline NugetAnalyzer.BLL.Test/Services/RepositoryServiceTests.cs]: consider move it to OneTimeSetup

**Review by AntsiferauGodel — CHANGES_REQUESTED** [review]

**Review by maximKalinov — APPROVED** [review]

File: NugetAnalyzer.BLL.Test/Services/RepositoryServiceTests.cs [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+                    {
+                        new Project
+                        {
+                            Name = "TestProject1",
+                            Packages = new List<Models.Packages.Package>
+                            {
+                                new Models.Packages.Package
+                                {
+                                    Name = "TestPackage",
+                                    Version = "1.1.1.1"
```
Comment (pipalalala) [inline NugetAnalyzer.BLL.Test/Services/RepositoryServiceTests.cs]: Pls move strings to constants

**Review by pipalalala — CHANGES_REQUESTED** [review]

File: NugetAnalyzer.BLL.Test/Services/RepositoryServiceTests.cs [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+                                    new Models.Packages.Package
+                                    {
+                                        Name="AnotherTestPackage",
+                                        Version = VersionString
+                                    }
+                                }
+                            },
+                            new Project
+                            {
+                                Name="TestProject2",
```
Comment (AntsiferauGodel) [inline NugetAnalyzer.BLL.Test/Services/RepositoryServiceTests.cs]: whitespaces?

**Review by AntsiferauGodel — CHANGES_REQUESTED** [review]

File: NugetAnalyzer.BLL.Test/Services/RepositoryServiceTests.cs [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+                        Projects = new List<Project>
+                        {
+                            new Project
+                            {
+                                Name = "TestProject1",
+                                Packages = new List<Models.Packages.Package>
+                                {
+                                    new Models.Packages.Package
+                                    {
+                                        Name = "TestPackage",
```
Comment (pipalalala) [inline NugetAnalyzer.BLL.Test/Services/RepositoryServiceTests.cs]: Same

**Review by pipalalala — CHANGES_REQUESTED** [review]

**Review by AntsiferauGodel — APPROVED** [review]
Seems ok

**Review by pipalalala — APPROVED** [review]

**Pull request merged**